airbyte-integrations/connectors/source-dixa/source_dixa/spec.json

```diff
@@ -19,7 +19,7 @@
         "examples": ["YYYY-MM-DD"]
       },
       "batch_size": {
-        "type": "string",
+        "type": "integer",
         "description": "Number of days to batch into one request. Max 31.",
         "pattern": "^[0-9]{1,2}$",
```

Comment:
what is default value? this attributes is not mandatory.

Reply:
There is no default value, unless we set it up "as default value", this is the batch_size parameter for the stream_slicing to produce the slices with the `batch_size` as days between each period. From my point of view `default is 31`, as the maximum available. The full range is: 1 - 31, where 1 is 1 day size, 31 - month of size.

From official "Conversation_Export" stream documentation:
```
https://support.dixa.help/en/articles/174-export-conversations-via-api
There are no offset or limit pagination rules for the endpoint, limits on queries are instead defined by time. In other words, no query can span more than 1 month (31 days). Here, the request must have at least one _after and one _before parameter. There are no other query limitations. 
```

From the Airbyte documentation: 
```
https://docs.airbyte.io/integrations/sources/dixa

When using the connector, keep in mind that increasing the batch_size parameter will decrease the number of requests sent to the API, but increase the response and processing time.
```